services/wallet/router/router.go

```diff
@@ -132,14 +131,12 @@ func newSuggestedRoutes(
 	amountIn *big.Int,
 	candidates routes.Route,
 	fromLockedAmount map[uint64]*hexutil.Big,
-	tokenPrice float64,
-	nativeChainTokenPrice float64,
+	updatedPrices map[string]float64,
```

Comment:
@Khushboo-dev-cpp could you please use `input *requests.RouteInputParams` instead of `uuid`, `amountIn` and `fromLockedAmount`?

```golang
func newSuggestedRoutes(input *requests.RouteInputParams, candidates routes.Route, updatedPrices map[string]float64) (*SuggestedRoutes, []routes.Route) {
```

Reply:
done :)